tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-minversion = 1.4.2
+minversion = 3.20.1
```

Comment:
Why does this need to change? This is different then cisco.ios, which is working.